components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -89,6 +125,53 @@ export default class AppsFormSelectField extends React.PureComponent<Props, Stat
                     value={value}
                     onChange={this.onChange as any} // types are not working correctly for multiselect
                     isDisabled={field.readonly}
+                    components={commonComponents}
+                    {...commonProps}
+                />
+            </div>
+        );
+    }
+
+    renderUserSelect() {
+        const {field} = this.props;
+        const placeholder = field.hint || '';
+        const value = this.props.value;
+
+        return (
+            <div className={'react-select'}>
+                <AsyncSelect
+                    id={`MultiInput_${field.name}`}
```

Comment:
if you are using several field props, I think it's easier to read if you get those before, for example on line 136.

We can control the field properties better that way too.

```typescript
const {multiselect, name, readonly, hint} = field;
```